lint.go

```diff
@@ -1159,11 +1159,17 @@ func (f *file) lintErrors() {
 	}
 }
 
-func lintCapAndPunct(s string) (isCap, isPunct bool) {
+func lintCapAndPunct(s string) (isCap, isPunct, isNewline bool) {
 	first, firstN := utf8.DecodeRuneInString(s)
 	last, _ := utf8.DecodeLastRuneInString(s)
 	isPunct = last == '.' || last == ':' || last == '!'
+	isNewline = last == '\n'
 	isCap = unicode.IsUpper(first)
+	if isNewline && strings.Index(s, "\n") != len(s)-1 {
+		// Don't flag strings that contain more than one newline. They
```

Comment:
This special exception to the newline rule is suspicious to me. Data you could pull up for this would be helpful.